.github/workflows/fips-old.yml

```diff
@@ -0,0 +1,67 @@
+# Copyright 2022 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+name: Provider old versions compat
+on: [pull_request, push]
+
+permissions:
+  contents: read
+
+jobs:
+  fips-provider-legacy:
+    strategy:
+      matrix:
+        module: [
+          {
+            url: "https://www.openssl.org/source/old/3.0/openssl-3.0.0.tar.gz",
+            tgz: openssl-3.0.0.tar.gz,
+            dir: openssl-3.0.0
+          },
+        ]
+    runs-on: ubuntu-latest
+    steps:
+      - name: create directory
+        run: mkdir ./current
+      - uses: actions/checkout@v2
+        with:
+          path: current
+      - name: download
+        run: wget ${{ matrix.module.url }}
+      - name: unpack
+        run: tar xzf ${{ matrix.module.tgz }}
+      - name: localegen
+        run: sudo locale-gen tr_TR.UTF-8
+      - name: config current
+        run: ./config enable-shared enable-fips
+        working-directory: ./current
+      - name: config dump current
+        run: ./configdata.pm --dump
+        working-directory: ./current
+      - name: make current
+        run: make -s -j4
+        working-directory: ./current
+      - name: make test current
+        run: make test HARNESS_JOBS=${HARNESS_JOBS:-4}
+        working-directory: ./current
+      - name: config module
+        run: ./config enable-shared enable-fips
+        working-directory: ${{ matrix.module.dir }}
```

Comment:
isnt this supposed to be ./current + moduledir?

Reply:
Ahh ok it sits at same level as current.

Reply:
Correct.